src/elements/common/base.scss

```diff
@@ -1,19 +1,19 @@
 @import './variables';
-@import '~box-react-ui/lib/styles/common/buttons';
-@import '~box-react-ui/lib/styles/common/inline-notifications';
-@import '~box-react-ui/lib/styles/modifiers/accessibility';
-@import '~box-react-ui/lib/styles/common/overlay';
-@import '~box-react-ui/lib/styles/common/pills';
-@import '~box-react-ui/lib/components/pill-selector-dropdown/PillSelector';
-@import '~box-react-ui/lib/components/notification/Notification';
+@import '../../styles/common/buttons';
```

Comment:
Seems like this should be `@import './buttons';`, along with anything else that's an immediate neighbor.

Reply:
This is inside the elements folder, not next to buttons?